api/providers/views.py

```diff
@@ -684,7 +684,9 @@ class RegistrationProviderSchemaList(JSONAPIBaseView, generics.ListAPIView, List
     serializer_class = RegistrationSchemaSerializer
 
     def get_default_queryset(self):
-        return self.get_provider().schemas.get_latest_versions(request=self.request, invisible=True).filter(active=True)
+        provider = self.get_provider()
+        default_schema_id = provider.default_schema.id if provider.default_schema else None
+        return provider.schemas.get_latest_versions(request=self.request, invisible=True, default_schema_id=default_schema_id)
```

Comment:
I was going to suggest doing the default ordering here instead of in the metaschema class in order to help preserve encapsulation, but that's not an option because "Each order_by() call will clear any previous ordering."

Reply:
Yeah I REALLY wanted to do it here. The only way I could is if I pretty much rewrote the `get_latest_versions` query here....